README.md

```diff
@@ -74,6 +81,7 @@ nexmo.creds.application-id=application-id
 nexmo.creds.private-key-contents=contents-of-the-key
 ```
 
+<a name="customization"></a>
```

Comment:
HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

Reply:
Modified